src/test/java/com/google/devtools/build/lib/remote/DiskCacheIntegrationTest.java

```diff
@@ -212,6 +212,27 @@ public void remoteExecWithDiskCache_hitDiskCache() throws Exception {
     events.assertContainsInfo("2 disk cache hit");
   }
 
+  @Test
+  public void remoteExecWithDiskCache_resetDiskCache() throws Exception {
```

Comment:
It's good that you added a integration test! However, I think this test is redundant - the scenario is test case tries to test is when the local disk cache directory is empty and we do a clean build. This is exactly the same as `remoteExecWithDiskCache_hitDiskCache`. WDTY?

Reply:
Whatever tests existed didn't catch the bug we had though.

Reply:
You're right, removed.